testdata/project-v3-config/PROJECT

```diff
@@ -16,6 +16,8 @@ resources:
   version: v1
   webhooks:
     defaulting: true
+    spokes:
+    - ""
```

Comment:
@momom-i

> @camilamacedo86 @varshaprasad96
> https://github.com/kubernetes-sigs/kubebuilder/pull/3058#discussion_r1012768842
> As you said before, I think we can just take the boolean option whether it is hub or spoke and create Hub(), ConvertTo() or ConvertFrom() template. For instance kubebuilder create webhook --version v1 --hub.
> What do you think?

Note that `kubebuilder create webhook --version v1 --hub.` The version is for teh webhook and not spoke, So, why are you adding an version to spoke? Look how the others options for webhooks are tracked. Example `conversion: true` . So, it shows for me that should be a boolean.

Reply:
@camilamacedo86 
I agree with you. And I've not changed yet since [@/varshaprasad96' PR](https://github.com/kubernetes-sigs/kubebuilder/pull/2084). I just fixed the points where the reviewers mentioned in that PR.
And I'd like to confirm to you whether I can change this PR to take only one boolean option or not in advance because it's going to be a big change.
As I mentioned before, example usage I'm supposed is like `kubebuilder create webhook --version v1 --conversion --hub` or `kubebuilder create webhook --version v1 --conversion --spoke.`

If user entered  `kubebuilder create webhook --version v1 --hub`(it implies conversion is true) , it will create `Hub()` skeleton code under `project/api/v1/{resource}_conversion.go`.
If user entered  `kubebuilder create webhook --version v2 --spoke` , it will create `ConvertTo()` or `ConvertFrom()`  skeleton code under `project/api/v2/{resource}_conversion.go`.